maquetado/scss/titles.scss

```diff
@@ -0,0 +1,25 @@
+$text-big: 24px;
```

Comment:
All this text styles sizes could be in a `fonts.scss` file. (not the `title-decoration` ones)